test/fixtures/integration/missing-fixture/test/tests.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <link rel="import" href="../../test-fixture/test-fixture.html">
+  <script src="/components/web-component-tester/browser.js"></script>
+</head>
+
+<body>
+  <script>
+    suite('suite', function () {
+      test('missing fixture', function () {
```

Comment:
Until `@polymer/test-fixture` package implements same logic, adding a test for that fails in Safari and Firefox because they appear to be resolving the `fixture()` function calls with `context.fixture` instead.